deps/jerry/jerry-ext/handle-scope/handle-scope-internal.h

```diff
@@ -76,6 +76,12 @@ jerryx_handle_scope_add_handle_to (jerryx_handle_t *handle, jerryx_handle_scope
 
 void
 jerryx_create_handle_in_scope (jerry_value_t jval, jerryx_handle_scope scope);
+
+jerryx_handle_scope_status
+jerryx_escape_handle_internal (jerryx_escapable_handle_scope scope,
+                           jerry_value_t escapee,
+                           jerry_value_t *result,
+                           bool should_promote);
```

Comment:
Alignments are required :(

Reply:
We have to turn of lint on jerry files :P

Reply:
Yea, just created an issue for this https://github.com/Rokid/ShadowNode/issues/258 :)